### PR TITLE
Delete MODULE_GRBL13.2

### PR DESCRIPTION
**Pull request by Tinyu-Zhao**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Tinyu-Zhao. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by Tinyu-Zhao** [comment]
Thank you.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/08/23 11:52:16 Loaded 6125 libraries from DB
Removing MODULE_GRBL13.2
Success!
```

**Pull request merged**
